modules/application/src/main/java/edu/northeastern/truthtree/assembler/StoriesAssembler.java

```diff
@@ -31,7 +31,7 @@ public StoriesAssembler(ObjectMapper mapper) {
     } catch (IOException e) {
       e.printStackTrace();
     }
-    return storyDTO.getData();
+    return storyDTO.getData().getData();
```

Comment:
I thought you renamed this to something else in previous pr? storyDTO variable isn't a `StoryDTO` obj in this case.

Reply:
I remember I changed it. Let make changes and commit once again.